### ISSUE TITLE
Styling applied to non-composable domain objects should be saved as properties of the layout.

### ISSUE THREAD

**Issue by shefalijoshi** [nasa/openmct#2779]
Styles on non-composable domain objects (like telemetry) should be saved in the current layout only, and not to the domain objects themselves. Currently, styling on a non-composable domain object in a layout styles the domain object, and all aliases of that domain object.

Editing styles on a composable domain object will apply and persist those styles as part of the domain object - consistent with our approach to sub-object editing.

So in short: non-composable domain object styles are treated like Layout drawing objects, and composable domain object styles are treated as sub-object editing.

**Comment by akhenry** [comment]
I think we're using compose-ability here is a proxy for what we really mean. I think (and please let me know if I'm missing something here) that the behavior we want is "if the selected thing is a display layout-specific view type (box, line, telemetry-view) then store the configuration on the Display Layout. Otherwise, store it on the selected object."

This can be achieved by inspecting the selection context from the inspector, and checking if `layoutItem.type !== 'subobject-view'`. I think that this mostly works? Happy to talk about it.
